catalogue/webapp/components/PhysicalItemDetails/PhysicalItemDetails.tsx

```diff
@@ -32,41 +43,45 @@ const Box = styled(Space).attrs<{ isCentered?: boolean }>(props => ({
     `
     align-self: center;
   `}
+
+  ${props => props.theme.media.medium`
+    margin-bottom: 0;
+  `}
 `;
 
 const Grid = styled.div<{ isArchive: boolean }>`
   ${props => props.theme.media[props.isArchive ? 'large' : 'medium']`
     display: grid;
-    grid-template-columns: max-content 130px 125px;
+    grid-template-columns: 160px 130px 125px;
     grid-column-gap: 25px;
-
-    ${Box}:first-child {
-      min-width: 240px;
-    }
   `}
 `;
 
 export type Props = {
   title: string;
   itemNote?: string;
-  locationAndShelfmark: string;
+  location: string;
+  shelfmark: string;
   accessMethod?: string;
   requestItemUrl?: string;
   accessNote?: string;
+  isLast: boolean;
```

Comment:
Given these are shared
```
Props = {
  //...
}
```

And then have
```
const PhysicalItemDetails: FunctionComponent<Props & { isLast: boolean }> = ({});
```

So you wouldn't have to remember to omit them [elsewhere](https://github.com/wellcomecollection/wellcomecollection.org/pull/6838/files#diff-218f5461bb26fcf8201bd315bbc5a853ff93a79e625e80046674ce8d2dcc76c4R33)

Reply:
Saying that - should the `PhysicalItem` be a type elsewhere that we can extend and share about? 

https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/model/catalogue.ts#L191?

Reply:
I've removed the `Omit` and changed the type as per your first suggestion – I think the props for PhysicalItemDetails aren't sufficiently reusable as a type in their own right (and I don't think it would make sense to use the pre-existing `PhysicalItem` type for them). What do you think?

Reply:
it still feels like PhysicalItem is not specific to the UI - but rather something returned by the API - is there a reason we're not jusdt using that?

Reply:
yep – I get you. I think because this work has been through so many iterations, I'd lost sight of the underlying data. I think you're right and we probably can/should just use the thing returned from the api. I'll put it in another pr.